classes/DirtEvent.sc

```diff
@@ -84,8 +84,9 @@ DirtEvent {
 
 		if (~unit == \c) { speed = speed * ~unitDuration * ~cps };
 
-		endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
-		if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		// endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
+		// if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		endSpeed = speed * (1.0 + accelerate);
```

Comment:
yep, in terms of terminology, this is right. What we could do is to supply two parameters:

- `accelerate` (default 0): delta-speed (either relative to cps or not)
- `hurry` (default 1): endSpeed derived from speed

then we can define in SuperDirt:
```
endSpeed =  speed * hurry + (accelerate + 1)
```
It's a bit like `harmonic` and `detune` for frequencies.